src/pages/Document/index.tsx

```diff
@@ -346,7 +367,7 @@ const FullProtocolButton = React.memo(function ({ url }) {
             }}
         >
             <Button
-                color="textSecondary"
+                color="secondary"
```

Comment:
?

Reply:
Text secndory not assignable to the type

Reply:
https://mui.com/material-ui/api/button/